src/ScriptCs.ComponentModel.Composition/ScriptCsCatalogOptions.cs

```diff
@@ -39,4 +44,4 @@ internal ScriptCsCatalogOptions OverridesNullByDefault()
             };
```

Comment:
A `KeepScriptsSeparated = KeepScriptsSeparated` line is missing to avoid the override of this property.